src/tests/test_buildercore_cfngen.py

```diff
@@ -122,27 +124,27 @@ def test_template_delta_includes_parts_of_cloudfront(self):
             "custom-subdomain.example.org"
         ]
         (delta_plus, delta_edit, delta_minus) = cfngen.template_delta(context)
-        self.assertEqual(delta_edit['Resources'].keys(), ['CloudFrontCDN', 'CloudFrontCDNDNS1'])
+        self.assertCountEqual(list(delta_edit['Resources'].keys()), ['CloudFrontCDN', 'CloudFrontCDNDNS1'])
         self.assertEqual(delta_edit['Resources']['CloudFrontCDNDNS1']['Properties']['Name'], 'custom-subdomain.example.org.')
-        self.assertEqual(delta_edit['Outputs'].keys(), [])
+        self.assertEqual(list(delta_edit['Outputs'].keys()), [])
 
     def test_template_delta_includes_parts_of_elb(self):
         "we want to update ELBs in place given how long it takes to recreate them"
         context = self._base_context('project-with-cluster')
         context['elb']['healthcheck']['protocol'] = 'tcp'
         (delta_plus, delta_edit, delta_minus) = cfngen.template_delta(context)
-        self.assertEqual(delta_edit['Resources'].keys(), ['ElasticLoadBalancer'])
+        self.assertEqual(list(delta_edit['Resources'].keys()), ['ElasticLoadBalancer'])
         self.assertEqual(delta_edit['Resources']['ElasticLoadBalancer']['Properties']['HealthCheck']['Target'], 'TCP:80')
-        self.assertEqual(delta_edit['Outputs'].keys(), [])
+        self.assertEqual(list(delta_edit['Outputs'].keys()), [])
 
     def test_template_delta_includes_elb_security_group(self):
         "for consistency with EC2 security groups"
         context = self._base_context('project-with-cluster')
         context['elb']['protocol'] = 'https'
         context['elb']['certificate'] = 'DUMMY_CERTIFICATE'
         (delta_plus, delta_edit, delta_minus) = cfngen.template_delta(context)
-        self.assertEqual(delta_edit['Resources'].keys(), ['ElasticLoadBalancer', 'ELBSecurityGroup'])
-        self.assertEqual(delta_edit['Outputs'].keys(), [])
+        self.assertCountEqual(list(delta_edit['Resources'].keys()), ['ElasticLoadBalancer', 'ELBSecurityGroup'])
```

Comment:
assertion method that wins the worst name in the year :)

Reply:
tell me about it.